### ISSUE TITLE
Compare with branch gives IndexError

### ISSUE THREAD

**Issue by magnli10** [jisaacks/GitGutter#219]
Hello,
My project has two branches. Attempting to diff the current(master) branch with the 2nd branch produce this error 

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 535, in run_
    return self.run()
  File "git_gutter_compare in .../Sublime Text 3/Installed Packages/GitGutter.sublime-package", line 15, in run
  File "git_gutter_compare in .../Sublime Text 3/Installed Packages/GitGutter.sublime-package", line 38, in commit_list
  File "git_gutter_compare in .../Sublime Text 3/Installed Packages/GitGutter.sublime-package", line 38, in <listcomp>
  File "git_gutter_compare in .../Sublime Text 3/Installed Packages/GitGutter.sublime-package", line 43, in parse_result
IndexError: list index out of range
```

osx : 10.10.2
st3 : 3080
gitgutter : 1.1.2

**Comment by rchl** [comment]
Output returned by git must not be what GitGutter expected. GitGutter runs this command:
```
git for-each-ref --sort=-committerdate --format=%(subject)\a%(refname)\a%(objectname) refs/heads/
```
If you still have this problem (unlikely after a year) then you could try running this command manually in the repo and paste the output.

**Comment by deathaxe** [comment]
This issue is fixed by commit 967f807 as empty git output will result in a message box.
